llvm/lib/Analysis/BasicAliasAnalysis.cpp

```diff
@@ -68,6 +68,10 @@ using namespace llvm;
 static cl::opt<bool> EnableRecPhiAnalysis("basic-aa-recphi", cl::Hidden,
                                           cl::init(true));
 
+/// Enable full analysis of PHI nodes.
+static cl::opt<bool> EnableFullPHIAnalysis("basic-aa-full-phi-analysis",
+                                           cl::Hidden, cl::init(true));
```

Comment:
nit: We initialize it to true by default and pass it from the driver as true.

Reply:
Oh, my mistake, fixed.